content/docs/studio/create-view.md

```diff
@@ -0,0 +1,54 @@
+# Create a View
+
+To create a new view for your repository, follow these steps.
+
+1. Click on `Add a View`. All the organizations that you have access to will be
+   listed. ![](/img/studio/create_view.png)
```

Comment:
```suggestion
To create a new view for your repository, follow these steps.

1. Sign into your [DVC Studio](https://viewer.iterative.ai/) dashboard. 
   ![](/img/studio/login_home.png)

2. Click on `+ Add a view`. All the organizations that you have access to will
   be listed.
   ![](/img/studio/create_view.png)
```

and so on for the remaining 3 steps (update numbers and add a line before each img)

Reply:
was already done